package.json

```diff
@@ -8,11 +8,11 @@
     "build": "npm-run-all clean:bundle test lint:build && babel-node Origami/scripts/build.js",
     "start": "babel-node server.js",
     "open:src": "babel-node server.js",
-    "autopep8": "autopep8 --ignore E901,E203,E201,E128,E202,W604,E226,W504 --recursive --in-place --pep8-passes 2000 api/* django_server/* docker/* docs/*{.py,.bat,Makefile} media/*",
+    "yapf": "yapf --in-place --recursive $(ls **/*.py) --exclude \"Origami/*\"",
```

Comment:
make this `yapf:fix`, this is somewhat misleading.

Reply:
Done in 92f8b9d908cc0e007170ef6fa5fbe39a05ecda3c.